components/footer/_footer.scss

```diff
@@ -1,13 +1,46 @@
 .footer {
   padding: 2rem 0;
   margin-top: 2rem;
-  background-color: $color-light-grey;
-  color: $color-very-dark-grey;
+  background-color: $color-very-dark-grey;
+  color: $color-light-grey;
+
+  // ul li { not required as list--bare is available
+  //   list-style-type: none;
+  // }
+  .footer-nav__heading {
+    margin: 2rem 0 1rem;
+  }
+  .dl__data {
```

Comment:
This doesn't belong here, it's an element of a `dl` which is nothing to do with the footer. I assume this was used to just use some styling from that element.